Blues/Peripheral/PeripheralProtocols.swift

```diff
@@ -29,6 +29,8 @@ public protocol PeripheralProtocol: class {
     func discover(services: [Identifier]?)
     
     func readRSSI()
+    
+    func updateAdvertisement(_ advertisement: Advertisement)
```

Comment:
I can't find a corresponding method on `CBPeripheral` that this would correspond to.
Could you elaborate on what this does and why it's needed?